lib/tasks/broadcasts.rake

```diff
@@ -0,0 +1,14 @@
+desc "Send Welcome Notifications"
+task broadcast_welcome_notification_flow: :environment do
+  # Should run once a day.
```

Comment:
let's move this in the description, so it appears when one runs `rails -T` to see available tasks